Find the common symbol.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+In this experiment each subject is given a card on which there appears a set of five (out of six possible) symbols. Each subject's card is different from all the others in that the symbol lacking is a different symbol in each case. Thus, in any set of five cards there is only one symbol in common. The task is for every subject to find the common symbol.
+
+# References
+Main paper: Leavitt, H. J. (1951). Some effects of certain communication patterns on group performance. The Journal of Abnormal and Social Psychology, 46(1), 38–50. https://doi.org/10.1037/h0057189
+
+# Stimuli
+## The visual components
+A card with a set of five out of six symbols as follows are presented to the subjects:
+
+![Alt text](/images/Find_the_common_symbol.jpg)
+
+# Procedure
+## Steps
+
+1. One hundred male undergraduates of M.I.T.,8 drawn from various classes at the Institute, served as subjects for these experiments.
+2. These 100 are split up into 20 groups of five men each. These 20 groups are then further subdivided so that five groups could be tested on each of four experimental patterns.
+3. In case the color or geographical position of one's work-space might affect one's behavior, positions are shifted for each new group.
+4. Each group is given 15 consecutive trials on one pattern, a process which required one session of about fifty minutes. These subjects are not used again. The order in which the patterns are used is randomized.
+  - Each subject is given a card on which there appear a set of five (out of six possible) symbols. Each subject's card is different from all the others in that the symbol lacking, the sixth one, is a different symbol in each case. Thus, in any set of five cards there is only one symbol in common.
+  - The problem is for every member to find the common symbol. To accomplish this, each member is allowed to communicate, by means of written messages, with those other members of the group to whom he has an open channel.
+  - A subject who has discovered the answer is allowed to pass the answer along.
+5. After a group has completed its 15 trials, and before members are permitted to talk with one another, each member is asked to fill out a questionnaire.
```

Comment:
When five lights (whether or not they were under the correct symbol), representing five different 5s, were lit, the observer called a halt to the trial. The observer could tell by a glance at the light panel whether (a) five different 5s had thrown their switches, (b) whether all five had decided on the same answer, and (c)whether the answer decided on was right or wrong.The same detailed instructions were given to all 5s. A preliminary series of four problems, in which each 5 was given all the information required for solution, was used. This was done to note the extent of differences among 5s in the time required to solve such problems